src/components/structures/auth/Registration.tsx

```diff
@@ -614,8 +614,8 @@ export default class Registration extends React.Component<IProps, IState> {
                     onServerConfigChange={this.state.doingUIAuth ? undefined : this.props.onServerConfigChange}
                 />
                 { this.renderRegisterComponent() }
-                { goBack }
-                { signIn }
+                <div className="mx_AuthBody_link_goBack">{ goBack }</div>
+                <div className="mx_AuthBody_link_signIn">{ signIn }</div>
```

Comment:
The margin styles you define for these wrappers can be added to the style of the `signIn` and `goBack` elements themselves - both use the class `mx_AuthBody_changeFlow`
It's unnecessary to introduce more wrapping elements.

Reply:
OK, I am going to fix the bug of the link with specifying additional rules

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8208/commits/dbdf83d3c2dc6ae0f0b7c8decda78542f480569d

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8208/commits/ca411f7757bbe4c6ddefeac942a9c33d548db621.

TODO: margin setting. See below.